### ISSUE TITLE
drkit instructions

### ISSUE THREAD

**Issue by petulla** [google-research/language#61]
hi 

i'm really interested in using drkit. i realize the repo is in the labs section, which comes with caveats. is it possible to get a set of minimal reproducible steps? the demo of the paper is really fascinating and it'd be great to be able to try using the codebase on a virtual kb.

**Comment by youssefavx** [comment]
Yeah would love to know this too. Is there a pre-trained model we could use? If not, would love steps to reproduce.

**Comment by krrishdholakia** [comment]
commenting for visibility - same here

**Comment by sd3ntato** [comment]
https://open-csr.github.io//methods/
